utils/user_input.py

```diff
@@ -1,19 +1,21 @@
-from tkinter import simpledialog, filedialog
-import tkinter as tk
+from tkinter import simpledialog, filedialog, Tk
```

Comment:
The full Tkinter libary is already added in the line above so no need to import specific modules again?

Reply:
Aangezien je dit al ge'resolved' hebt heb je het waarschijnlijk later gezien, maar die rode lijnen met een - ervoor heb ik dus vervangen door de groene lijn met een +.
Voor sommige wijzigingen is het handig om de 'changes' te zien. Soms is het handig om het volledige bestand gewoon even te openen (en dan zou je hier zien dat er op 1 regel 'import' gedaan wordt.